src/routers/router.ts

```diff
@@ -35,5 +37,6 @@ router.get('/developers/account', attachRedisClient, getDeveloper);
 
 router.get('/stats', validateAdminApiKey, attachRedisClient, getStats);
 router.post('/login', login);
+router.post('/logout', isAuthenticated, logout);
```

Comment:
we want to use verb form for middleware as much as possible

```suggestion
router.post('/logout', authenticate, logout);
```